site.js

```diff
@@ -532,15 +541,15 @@ app = {
                             url = 'bikelanes/' + laneData;
                             app.laneAjaxQueue(url, region);
                         } else {
-                            console.log('=== ALERT: Missing data for', region);
+                            app.showError('Missing data for' + region);
                         }
                     } else {
                         app.display.regionsArray = ['AL', 'AR', 'DC', 'MO', 'PG'];
                     }
                 }
             }
         }).fail(function(){
-            console.log('*** ajax fail T ***');
+            app.showError('AJAX failure');
```

Comment:
Could we change `AJAX failure` to something more user-friendly like, "Error: Unable to load data"?